packages/edge-login-ui-rn/src/actions/CreateAccountActions.ts

```diff
@@ -55,15 +55,14 @@ export function checkUsernameForAvailabilty(data: string) {
 }
 
 function isASCII(str: string) {
-  // eslint-disable-next-line no-control-regex
-  return /^[\x00-\x7F]*$/.test(str)
+  return /^[\x20-\x7E]*$/.test(str)
```

Comment:
For future reference, we like to see fixes like this as their own commit, not bundled with other work.